rbac/management/workspace/utils/access.py

```diff
@@ -29,9 +30,35 @@
 from management.utils import get_principal_from_request, roles_for_principal
 from rest_framework.serializers import ValidationError
 
+from rbac import settings
+
 logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
 
 
+def _log_v2_timing(timings, total_start, extra_fields, reason=None):
```

Comment:
do we wanna add principal user_id or something for correlation? so that we know which user it is causing issue